Paths update script

am-e opened the pull request:
Perl script to update Javascript and CSS paths in HTML.

am-e committed:
Project mapping, queries, ingest and browser initial

am-e committed:
Added project browser

am-e committed:
Removed query limit, Placed prod URLs

am-e committed:
Added Reporting Year facet

am-e committed:
Ingest project code cleanup

am-e committed:
Added comments to HTML, ReportingYear not Optional

am-e committed:
Updated mapping, ingest scripts with correct cardinality of some fields. Refactored HTML with handlebars

am-e committed:
Made Decsription optional in describeProject.rq

am-e committed:
Perl script to modify paths

am-e committed:
Removed Perl script

am-e committed:
Added paths update script

am-e committed:
Added paths update script

mrpatrickwest commented:
If I had to guess I'd say that you created this branch from your project browser branch? I would expect to see the script in here and not your project browser files.

mrpatrickwest commented on scripts/switch_paths.pl:
```diff
@@ -0,0 +1,22 @@
+$num_args = $#ARGV + 1;
+if ($num_args != 2) {
+    print "\nUsage: switch_dev_prod.pl dev/prod filename\n";
+    exit;
```
Add a print here to describe what the script does. For completeness.

mrpatrickwest commented on scripts/switch_paths.pl:
```diff
@@ -0,0 +1,22 @@
+$num_args = $#ARGV + 1;
+if ($num_args != 2) {
+    print "\nUsage: switch_dev_prod.pl dev/prod filename\n";
```
Thinking you mean dev|prod and not dev/prod

am-e committed:
Fixed typo, added script description print statement

am-e commented:
I had initially added the script under the project-browser, then realized the error of my ways and removed the script. I then created a new branch online, checked it out locally, added the script and pushed the changes.

zednis commented:
Hi Ahmed,  based on the branch network diagram (see Graphs on the right-hand side in github web UI) the Paths-update-script branch was branched off the project-browser branch and as such includes in it's history all the previous commits to the project-browser branch.

merging this branch to Master will also merge all the previous commits made to project-browser.

zednis commented:
My recommendation is we close this pull request without merging, delete the Paths-update-script branch, and create a new path-script branch based on the current master branch.  We will then be able to merge the path script into master fully independent of the project-browser updates.

am-e commented:
Yes, I agree. Sorry about that, I'll be more careful in the future. If we are all in agreement, I'll go ahead and implement what you suggested.

zednis commented:
:+1:

mrpatrickwest commented:
:+1:
